integration/optional-defaults-incorrect/terraform.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = "~> 1.7.0"
+  required_providers {
+    azurerm = {
+      version = ">= 3.98.0"
```

Comment:
Please change the order. I assume the order should be like below this.

azurerm = {
  source = "hashicorp/azurerm"
  version = ">= 3.97.0, < 3.99.0"
}